### ISSUE TITLE
[UWP][Accessibility]After uploading Incorrect file error message is not narrated by Screen Reader.

### ISSUE THREAD

**Issue by almedina-ms** [microsoft/AdaptiveCards#3881]
# Platform
* UWP

# Version of SDK
master

# Details

Repro Steps:
1. Launch the application.
2. New screen starts appearing. Navigate to the top header elements.
3. Now select "Save File" button. It will save the file.
4. Verify the issue.

Actual Result: 
After uploading Incorrect file error message is not narrated by Screen Reader.
Screen reader is not announcing anything while error message is displaying in screen.

Expected Result: 
After uploading Incorrect file error message should be narrated by Screen Reader.
